src/BuiltInTools/AspireService/AspireServerService.cs

```diff
@@ -412,22 +382,25 @@ private async Task SendMessageAsync(string dcpId, byte[] messageBytes, Cancellat
             return;
         }
 
+        var success = false;
         try
         {
-            using var cancelTokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, _shutdownCancellationTokenSource.Token,
-                                                                                          connection.HttpRequestAborted);
+            using var cancelTokenSource = CancellationTokenSource.CreateLinkedTokenSource(
+                cancellationToken, _shutdownCancellationTokenSource.Token, connection.HttpRequestAborted);
```

Comment:
This seems like a bit much per message.

Reply:
Suggestions?

Reply:
This might be the only thing you can do given the calling pattern... Ignore for now.